<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">left: 90;
top: 24;
bottom: 26;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 5px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(5, 16, 54);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>370</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>311</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">left: 24;
right: 29;
top: 24;
bottom: 42;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 5px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_12">
              <property name="styleSheet">
               <string notr="true">border: one;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_13">
              <property name="styleSheet">
               <string notr="true">border: one;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>RadioButton</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>RadioButton</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_15">
              <property name="styleSheet">
               <string notr="true">border: one;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QTextBrowser" name="textBrowser">
                 <property name="minimumSize">
                  <size>
                   <width>345</width>
                   <height>600</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>345</width>
                   <height>700</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: #0263BC;
color: #ffffff;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_14">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">border: one;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="minimumSize">
                  <size>
                   <width>157</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>157</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QPushButton{
color: #ffffff;
box-shadow: 0px 0px 0px 2px #9fb4f2;
background-color: rgb(14, 63, 89);
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 10px;
}
QPushButton:hover{
background: #0E3C56;

}
QPushButton:pressed{
background: rgb(18, 81, 117);
}
</string>
                 </property>
                 <property name="text">
                  <string>Отобразить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="minimumSize">
                  <size>
                   <width>157</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>157</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QPushButton{
color: #ffffff;
box-shadow: 0px 0px 0px 2px #9fb4f2;
background-color: rgb(14, 63, 89);
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 10px;
}
QPushButton:hover{
background: #0E3C56;

}
QPushButton:pressed{
background: rgb(18, 81, 117);
}
</string>
                 </property>
                 <property name="text">
                  <string>Очистить окно</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>1090</width>
          <height>900</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="minimumSize">
            <size>
             <width>1088</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1088</width>
             <height>72</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">left: 90;
top: 24;
bottom: 26;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 5px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>800</width>
                <height>42</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
left: 90;
right: 90;
top: 15;
bottom: 15;

font-style: normal;
font-weight: normal;
fint-size: 24px;
line-height: 42px;

text-align: center;
border: none;
color: #FFFFFF;
</string>
              </property>
              <property name="text">
               <string>Главное окно управления и отображения состояния</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="styleSheet">
            <string notr="true">left: 990;
top: 24;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 5px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QTableView" name="tableView">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>10</y>
              <width>256</width>
              <height>192</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="styleSheet">
            <string notr="true">left: 990;
top: 24;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 5px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QFrame" name="frame_16">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>55</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_17">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_18">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>284</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>284</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">position: absolute;
top: 42px;
left: 20;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;


border-top-left-radius: 150;
border-top-right-radius: 150;
border-bottom-left-radius: 5px;
border-bottom-right-radius: 5px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QFrame" name="frame_10">
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButton_5">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>150</horstretch>
                   <verstretch>150</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
font-style: normal;
font-weight: normal;
font-size: 42px;
line-height: 52px;
text-align: center;
color: #ffffff;
box-shadow: 0px 0px 0px 2px #AA2121;
background-color: rgb(170, 5, 5);
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 75;
}
QPushButton:hover{
background: rgb(229, 0, 0);

}
QPushButton:pressed{
background: #AA2121;
}
</string>
                 </property>
                 <property name="text">
                  <string>СТАРТ</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_11">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="minimumSize">
                  <size>
                   <width>157</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QPushButton{
margin-right: 5;
color: #ffffff;
box-shadow: 0px 0px 0px 2px #9fb4f2;
background-color: rgb(14, 63, 89);
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 10px;
}
QPushButton:hover{
background: #0E3C56;

}
QPushButton:pressed{
background: rgb(18, 81, 117);
}
</string>
                 </property>
                 <property name="text">
                  <string>Загрузить
конфиг</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="minimumSize">
                  <size>
                   <width>157</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QPushButton{
margin-left: 5;
color: #ffffff;
box-shadow: 0px 0px 0px 2px #9fb4f2;
background-color: rgb(14, 63, 89);
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 10px;
}
QPushButton:hover{
background: #0E3C56;

}
QPushButton:pressed{
background: rgb(18, 81, 117);
}
</string>
                 </property>
                 <property name="text">
                  <string>Отобразить
лог файл</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">left: 24;
top: 24;
bottom: 42;

background: #125175;
border: 1px solid #000000;
box-sizing: border-box;
border-radius: 5px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
